proto/README.md

```diff
@@ -1,7 +1,18 @@
+## Update
```

Comment:
@zvolin  Which path do you prefer `proto/vendor/README.md` or `proto/README.md`?

The problem with `proto/README.md` is that it would not be nice when we publish the crate.
The problem with `proto/vendor/README.md` is that is not noticable.